lychee-lib/src/client.rs

```diff
@@ -20,17 +23,17 @@ const DEFAULT_USER_AGENT: &str = concat!("lychee/", env!("CARGO_PKG_VERSION"));
 
 #[derive(Debug, Clone)]
 pub struct Client {
-    /// The underlying reqwest client instance that handles the HTTP requests
+    /// Underlying reqwest client instance that handles the HTTP requests.
     reqwest_client: reqwest::Client,
-    /// Github API client
-    github: Option<Github>,
-    /// Filtered domain handling
+    /// Github API token.
+    github_token: Option<Github>,
```

Comment:
Wait, shouldn't that actually be `github_client`? After all, we call `Github::new(user_agent, Credentials::Token(token)` further down below and store that as `github_token`. 🤔

Reply:
Yep. Fixed